### PR TITLE
Add: Get user information

### PR DESCRIPTION
**Pull request by yvolkan**
Get user information

### CODE REVIEW

**Comment by vinkla** [comment]
Thanks for the pull request! Please add tests, documentation and always add a description to your pull requests.

**Comment by yvolkan** [comment]
Thanks for the comment. I did it.

File: README.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ $instagram = new Instagram('your-access-token');
 
 // Fetch the user's recent media feed.
 $instagram->get();
+
+// Get user information
+$instagram->me();
```
Comment (vinkla) [inline README.md]: Rename the function to `self()` instead to mimic the API endpoints.

**Comment by vinkla** [comment]
You should always to try to write a thorough description in your pull requests. Now you basically just copied and pasted the title.

**Comment by vinkla** [comment]
The code coverage is still below 100%, try to get it back to 100%.

**Comment by vinkla** [comment]
Please fix the syntax errors.

**Comment by yvolkan** [comment]
Thanks for the reply. Which the syntax errors?

**Comment by vinkla** [comment]
These: https://github.styleci.io/analyses/zOKGkl

**Comment by vinkla** [comment]
Good, then it just the code coverage left. We need to hit 100%.

https://codecov.io/gh/vinkla/instagram/src/ea21a732eb3ed7983907a21eca9e4abac6e58a8f/src/Instagram.php#L105...107

**Comment by yvolkan** [comment]
Thanks for helping me @vinkla. My first request to different users repository.

**Comment by vinkla** [comment]
Looks good @yvolkan! I'll probably do some minor changes but the new features will be released with the next version.

**Pull request merged**